procedures/MT/rotator/rotator_verification.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<p>So there is a "constant speed test" yet you say the CSC doesn't support it?</p><p>I'm confused.... is it being added? or is the test meant to fail?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
The CSC does not support the constant speed movement. Only EUI can do this. Therefore, this test case will need to use the EUI instead. Thanks!

Reply:
Improved the document in commit: [da2822c](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/da2822c02b133ed099461c12c208f882ed94bd6d). Thanks!